examples/website/i3s-picking/src/components/drop-down.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import styled from 'styled-components';
+import {Color, Font} from './styles';
+
+export const DropDownStyle = `
```

Comment:
Is it reusable? I'd include it to the styled component

Reply:
For now it's used in the DropDown component only

Reply:
You don't need a constant if it is not reusable. Just include styles into the component